### ISSUE TITLE
doesn't work on Safari

### ISSUE THREAD

**Issue by vesper8** [hilongjw/vue-lazyload#366]
Have tried v-lazy on Safari Mobile and it just doesn't work

It may work on later versions, but it doesn't on most versions including the version on the Xcode simulator

My question is.. is there a way to have a fallback for Safari that doesn't require a lot of extra if/else code ? Otherwise can't use this package at all as it affects a too large number of users

**Comment by cizza** [comment]
Yup, I just came across this issue, where lazyload doesn't work on any iPhones, iPads. The end result in my case is that the images won't load at all. The lazyload="error".

I don't like to moan at people who create open source software for bugs but this in my opinion is really big and renders this plugin absolutely useless for production. Please fix this.

**Comment by Solant** [comment]
@vesper8 
Checked on demo page on iOS 10 and it works fine
Make sure that you don't use https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API , as it's not supported by older iOS versions

**Comment by harmakit** [comment]
@vesper8 
Sure. If you want to use IntersectionObserver you should require it in your project 
`require('intersection-observer');`
Get it here: https://www.npmjs.com/package/intersection-observer

Or, you can just disable observer in vue-lazyload settings
```
Vue.use(VueLazyload, {
  observer: false // btw it's false by default
})
```

**Comment by adndyxu** [comment]
env: safari in ios 12.4.6 and above.
When the list is long,lazyload doesn't work very well,it only render serveral images ahead.
When u continue scroll,the image renders only if u set a specific height and overflow-y:auto on the list wrapper or waiting for the scroll stopped then tap or scroll the page.
In my case,there are some animate effects,so I can't set height for the wrapper.

**Comment by highoncarbs** [comment]
Any update ? For older versions ?
